app/components/publications/publication/case/info/publication-case-info-panel.js

```diff
@@ -0,0 +1,271 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { isBlank } from '@ember/utils';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { timeout, task, restartableTask } from 'ember-concurrency';
+import moment from 'moment';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class PublicationsPublicationCaseInfoPanelComponent extends Component {
+  @service store;
+  @service intl;
+  @service publicationService;
+
+  @tracked isInEditMode;
+
+  @tracked isViaCouncilOfMinisters;
+
+  @tracked isUrgent;
+
+  @tracked publicationNumberErrorKey;
+  @tracked publicationNumber;
+  @tracked publicationNumberSuffix;
+
+  @tracked numacNumbers;
+  numacNumbersToDelete;
+
+  @tracked decisionDate;
+  @tracked openingDate;
+  @tracked publicationDueDate;
```

Comment:
Is het nodig om van al die properties een lokale `@tracked` property bij te houden? Maken we het daarmee niet complexer dan nodig en is het niet makkelijker om gewoon rechtstreeks op de ember-data models te werken? Voor sommige zaken, zoals die numacNummers, zal het inderdaad wel handiger zijn, maar voor voor bijvoorbeeld de dates en `isUrgent` lijkt het mij een onnodige extra laag complexiteit die wordt toegevoegd.

Edit: ik heb me pas later in de review beseft dat de properties van `this.args.publicationFlow` inderdaad gecached moeten worden omdat ze anders misschien mee opgeslagen zullen worden bij `save` in het Opmerkingen-panel. Voor die cases zou ik dat dan expliciet documenteren dat dat de reden is.